codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -36,19 +36,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.zip.Checksum;
 
-import static io.netty.handler.codec.compression.Lz4Constants.BLOCK_TYPE_COMPRESSED;
-import static io.netty.handler.codec.compression.Lz4Constants.BLOCK_TYPE_NON_COMPRESSED;
-import static io.netty.handler.codec.compression.Lz4Constants.CHECKSUM_OFFSET;
-import static io.netty.handler.codec.compression.Lz4Constants.COMPRESSED_LENGTH_OFFSET;
-import static io.netty.handler.codec.compression.Lz4Constants.COMPRESSION_LEVEL_BASE;
-import static io.netty.handler.codec.compression.Lz4Constants.DECOMPRESSED_LENGTH_OFFSET;
-import static io.netty.handler.codec.compression.Lz4Constants.DEFAULT_BLOCK_SIZE;
-import static io.netty.handler.codec.compression.Lz4Constants.DEFAULT_SEED;
-import static io.netty.handler.codec.compression.Lz4Constants.HEADER_LENGTH;
-import static io.netty.handler.codec.compression.Lz4Constants.MAGIC_NUMBER;
-import static io.netty.handler.codec.compression.Lz4Constants.MAX_BLOCK_SIZE;
-import static io.netty.handler.codec.compression.Lz4Constants.MIN_BLOCK_SIZE;
-import static io.netty.handler.codec.compression.Lz4Constants.TOKEN_OFFSET;
+import static io.netty.handler.codec.compression.Lz4Constants.*;
```

Comment:
The wildcard import is not allowed. Revert.

Reply:
I used netty code style, but it changed after importing with shortcut key.

Reply:
@hyperxpro fixed.